app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class CommentsController < ApplicationController
+      skip_before_action :verify_authenticity_token, only: [:create]
+
+      def index
+        # Trying without instance variables
+        post_comments = Comment.where(user_id: params[:user_id], post_id: params[:post_id])
+        render json: post_comments
+      end
+
+      def create
+        comment = Comment.new(comment_params)
+        if comment.save
+          render json: comment, status: :created
+        else
+          render json: { error: comment.errors.full_messages }, status: :unprocessable_entity
+        end
+      end
```

Comment:
- [ ] Great work so far! I've noticed that the API endpoint for adding a comment to a posts is not working. The expected route for this operation is **POST :** `/users/:user_id/posts/:id/comments`. I've attempted to create a comment multiple times, but unfortunately, it doesn't seem to be functioning as expected.

  _Since both of you are currently unavailable, I'm unable to approve this at the moment. If I'm mistaken, kindly provide a screenshot demonstrating a successful comment creation in this PR. This will help in the next code review to confirm and approve if everything is indeed in order._ 👍🏻

Reply:
Thank you very much sir for the review, attached is a screenshot of adding a comment
<img width="1192" alt="Screenshot 2023-11-10 at 10 56 07 PM" src="https://github.com/diarisdiakite/Blog-app/assets/36508436/93643b8e-6de8-4103-ae65-3be4532f65ac">